src/app/qgisapp.cpp

```diff
@@ -4061,6 +4061,8 @@ void QgisApp::askUserForGDALSublayers( QgsRasterLayer *layer )
 
   // We initialize a selection dialog and display it.
   QgsSublayersDialog chooseSublayersDialog( QgsSublayersDialog::Gdal, QStringLiteral( "gdal" ), this );
+  bool display = true;
```

Comment:
Is there a reason to add this variable? I think it's only used on the next line.